vercel.json

```diff
@@ -1,4 +1,6 @@
 {
+  "$schema": "https://openapi.vercel.sh/vercel.json",
+  "buildCommand": "sh vercel.sh",
```

Comment:
Is there no way to avoid an additional shell script for this?

Reply:
i tired to be more explicit with it, we can always return early in `generate-md-exports` depending on [`VERCEL_ENV`](https://vercel.com/docs/environment-variables/system-environment-variables#VERCEL_ENV). This was their recommended approach https://vercel.com/guides/dynamic-build-commands

Reply:
I prefer that much more than this as it's less abstraction. Up to you though